site/search/SearchPanel.tsx

```diff
@@ -128,17 +136,152 @@ function ChartHit({ hit }: { hit: IChartHit }) {
     )
 }
 
-function ExplorerHit({ hit }: { hit: IExplorerHit }) {
+interface ExplorerViewHitWithPosition extends IExplorerViewHit {
```

Comment:
I think this is a regression not from this PR, but it seems like `data-algolia-position` is now only local to each section, i.e. posts start at 1, charts start at 1, etc.

charts were meant to start at `numberOfPosts + 1`

Reply:
I think that's what it was meant to do, right?
`data-algolia-position` is the _local_ position, and `globalPosition` gets computed in the click handler on its own (and only gets sent to GTM, not to Algolia).